libraries/plugins/webserver/include/sophiatx/plugins/webserver/webserver_plugin.hpp

```diff
@@ -30,7 +30,14 @@ using namespace appbase;
 class webserver_plugin : public appbase::plugin< webserver_plugin >
 {
    public:
-      webserver_plugin();
+      static std::shared_ptr<webserver_plugin> &get_plugin() {
+         static std::shared_ptr<webserver_plugin> instance(new webserver_plugin);
```

Comment:
make_shared

Reply:
As above, make_shared does not see the private constructor of webserver plugin